src/current/v22.2/licensing-faqs.md

```diff
@@ -85,7 +85,7 @@ See [Enterprise Features](enterprise-licensing.html) for more information.
 All CockroachDB code is included in the same binary. No license key is required to access BSL and CCL (Free) features. To access CCL (Paid) features, users have two options:
 
 - An **Enterprise license** enables you to use CockroachDB Enterprise features for longer periods (one year or more). To upgrade to an Enterprise license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
-- A **Trial license** enables you to try out CockroachDB Enterprise features for 30 days for free. To obtain a Trial license, fill out [the registration form](https://www.cockroachlabs.com/get-cockroachdb/enterprise/) and receive your trial license via email within a few minutes.
+- A **Trial license** enables you to try out CockroachDB Enterprise features for 30 days for free. To obtain a Trial license, <a href="mailto:sales@cockroachlabs.com">contact Sales</a>.
```

Comment:
I wonder if it would be helpful to also set an email subject on the `mailto:` link, like this:
```suggestion
- A **Trial license** enables you to try out CockroachDB Enterprise features for 30 days for free. To obtain a Trial license, <a href="mailto:sales@cockroachlabs.com?subject=Request&nbsp;free&nbsp;trial">contact Sales</a>.
```

Reply:
Nice idea. It's probably worth separating that from this PR and asking sales first. Created an internal [issue](https://cockroachlabs.atlassian.net/browse/DOC-9057).